server.js

```diff
@@ -90,23 +90,42 @@ if (process.env.NODE_ENV === 'production') {
     'styles',
     'manifest.json'
   ))
-  hbs.registerHelper('static-asset', (type, name) => {
+  const imagesManifest = require(path.join(
+    __dirname,
+    'precompiled',
+    'images',
+    'manifest.json'
+  ))
+  hbs.registerHelper('static-asset', (type, ...parts) => {
+    // `parts` should be at minimum [name, function]
+    // but it could also be [part1, part2, part3, function ]
+    // if we want to link to dynamic images
```

Comment:
What does `function` mean in this context? Is it the file extension?

Reply:
No, it's actually a function :)